pkg/controller/mpi_job_controller_test.go

```diff
@@ -504,6 +504,18 @@ func TestDoNothingWithInvalidMPIJob(t *testing.T) {
 	f.run(getKey(mpiJob, t))
 }
 
+func TestDoNothingWithMPIJobManagedExternally(t *testing.T) {
+	f := newFixture(t, "")
+	var replicas int32 = 1
+	startTime := metav1.Now()
+	completionTime := metav1.Now()
+	mpiJob := newMPIJob("test", &replicas, &startTime, &completionTime)
+	mpiJob.Spec.MPIImplementation = kubeflow.MPIImplementationOpenMPI
+	mpiJob.Spec.RunPolicy.ManagedBy = ptr.To(kubeflow.MultiKueueController)
+	f.setUpMPIJob(mpiJob)
+	f.run(getKey(mpiJob, t))
+}
+
```

Comment:
Additionally, could you add if the MPIJobs with `kubeflow.org/mpi-operator` are reconciled by the mpi-operator?

Reply:
I could add the mpi-operator in `newMPIJobCommon`, so all other tests verify that, although they should already as this is default value
```
	if policy.ManagedBy == nil {
		policy.ManagedBy = ptr.To(KubeflowJobController)
	}
```

Reply:
I figured that proving no kubeActions should be sufficient, wdyt?